dali/pipeline/operators/python_function/util/copy_with_stride.cc

```diff
@@ -40,39 +40,42 @@ inline void CopyVecStatic(uint8 *output, const uint8 *input, size_t elems, size_
 }
 
 inline void CopyWithStrideHelper(void *output, const void *input,
-                                 const std::vector<Index> &in_strides,
-                                 const std::vector<Index> &out_strides,
-                                 const std::vector<Index> &shape,
+                                 const Index *in_strides,
+                                 const Index *out_strides,
+                                 const Index *shape,
+                                 Index ndim,
                                  Index dim, Index deepest_contiguous) {
   auto out_ptr = reinterpret_cast<uint8*>(output);
   auto in_ptr = reinterpret_cast<const uint8*>(input);
+  const auto item_size = out_strides[ndim - 1];
   if (dim == deepest_contiguous) {
     std::memcpy(out_ptr, in_ptr,
```

Comment:
For the near future: in device code, a simple loop outperforms `memcpy`. and `std::memcpy` is not available.